src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -101,7 +103,12 @@ public function flush()
         $this->data = array();
     }
 
-    public function push($stat, $value, $time = null)
+    /**
+     * @param string $stat
+     * @param string $value
+     * @param int|null $time
+     */
+    private function pushStat($stat, $value, $time = null)
```

Comment:
@lyrixx 

> it breaks BC for nothing

It doesn't. Flagging as `@deprecated`will raise `E_USER_WARNING` in internal calls. Switching it directly to `private` will break BC. 

> Anyway, I want do keep this method in the public API.

Then we should add another interface then... same issue as the `GaugeableInterface` (that I should rename `Gaugeable` btw).

> More over, Graphite handler is not meant to be used in production (better use statsd).

Maybe it shouldn't be part of the project then :) Or under another NS?

@stof I hear you now about the multiple PRs... I believed one change per commit would be enough but this PR seems to have WAY too many changes to be processed at once ;)

However, I believe the new interface and the PHPDoc fix should be part of the same PR, don't you think?